### DISCUSSION TITLE
openAI is giving 404 errors

### DISCUSSION THREAD

**Discussion by bwhittaker69** [dotnet/ResXResourceManager#667] [General]
Hi, great product - looking for some help with the interface for OpenAI configuration...

i am getting 404 errrors .. where can i set the OpenAI endpoint or troubleshoot?

Thank you

![image](https://github.com/user-attachments/assets/b961d151-85f3-45b1-ac1f-ddec1f8ce15f)
![image](https://github.com/user-attachments/assets/bb42d6d9-e863-44a6-9ae9-1f2cdce45faa)

**Comment by tom-englert** [comment]
@Gr3q can you help with this?

**Comment by tom-englert** [comment]
https://github.com/dotnet/ResXResourceManager/blob/master/src%2FResXManager.Translators%2FOpenAITranslator.cs

**Comment by Gr3q** [comment]
The URL is set to be `https://api.openai.com` here https://github.com/dotnet/ResXResourceManager/blob/master/src%2FResXManager.Translators%2FOpenAITranslator.cs#L55

The full url is `https://api.openai.com/v1/chat/completions`

I tested it, your exact config shown (minus the key) does work in the extension and in the standlone app.

* What is your target language?
* Do you have any keys, if so can you provide the first one in the lsit with it's value if possible?

**Comment by Gr3q** [comment]
Your account needs to be at least Tier 1 to use gpt-4-turbo: https://platform.openai.com/docs/guides/rate-limits/tier-1-rate-limits
